elotl/utils/config.py

```diff
@@ -0,0 +1,9 @@
+#
```

Comment:
Do you think this file might be better on a macrolanguage-by-macrolanguage basis, e.g. live in `elotl/nahuatl` and `elotl/huave`? Then it could just be `DEFAULT_LANG_CODE` and `SUPPORTED_LANG_CODES`.